doc/helpers/ancestor-itemscontrol-binding.md

```diff
@@ -81,25 +81,45 @@ xmlns:utu="using:Uno.Toolkit.UI"
-<Border Tag="From Border">
-    <ListView ItemsSource="{Binding Items}" Tag="From ListView">
-        <ListView.ItemTemplate>
-            <DataTemplate>
-                <StackPanel>
-                    <TextBlock>Current item DataContext: <Run Text="{Binding}" /></TextBlock>
-                    <TextBlock>Current parent(ExampleModel) DataContext:
-                        <Run Text="{utu:AncestorBinding AncestorType=ListView, Path=DataContext.SomeText}" />
-                    </TextBlock>
-
-                    <TextBlock>Accessing property of a parent ListView:
-                        <Run Text="{utu:AncestorBinding AncestorType=ListView, Path=Tag}" />
-                    </TextBlock>
-                    <TextBlock>Accessing property of a parent Border:
-                        <Run Text="{utu:AncestorBinding AncestorType=Border, Path=Tag}" />
-                    </TextBlock>
-                <StackPanel>
-            </DataTemplate>
-        </ListView.ItemTemplate>
-    </ListView>
-</Border>
+<Page x:Class="AncestorBindingSample.MainPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:AncestorBindingSample"
+      xmlns:utu="using:Uno.Toolkit.UI"
+      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+      Tag="From Page">
+    <Grid RowSpacing="20"
+          HorizontalAlignment="Center">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <TextBlock Text="From DataContext:"
+                   FontSize="30"
+                   FontWeight="Bold">
+            <TextBlock.Inlines>
+                <Run Text="{Binding SomeText}" />
+            </TextBlock.Inlines>
+        </TextBlock>
+        <ListView ItemsSource="{Binding Items}"
+                  Grid.Row="1"
+                  Tag="From ListView">
+            <ListView.ItemTemplate>
+                <DataTemplate>
+                    <StackPanel>
+                        <TextBlock Text="Current item DataContext:" />
+                        <TextBlock FontWeight="Bold" Text="{Binding}" />
+                        <TextBlock Text="Current parent (ExampleModel) DataContext:" />
+                        <TextBlock FontWeight="Bold" Text="{utu:ItemsControlBinding Path=DataContext.SomeText}" />
+                        <TextBlock Text="Accessing property of a parent ListView:" />
+                        <TextBlock FontWeight="Bold" Text="{utu:ItemsControlBinding Path=Tag}" />
+                        <TextBlock Text="Accessing property of a Page:" />
+                        <TextBlock FontWeight="Bold" Text="{utu:AncestorBinding AncestorType=Page, Path=Tag}" />
+                    </StackPanel>
+                </DataTemplate>
+            </ListView.ItemTemplate>
+        </ListView>
+    </Grid>
+</Page>
```

Comment:
avoid dumping the full xaml here
just keep it short and straight to the point:
```suggestion
<Page Tag="From Page">
    <StackPanel>
        <TextBlock Text="From DataContext:" />
        <TextBlock Text="{Binding SomeText}" />

        <ListView ItemsSource="{Binding Items}" Tag="From ListView">
            <ListView.ItemTemplate>
                <DataTemplate>
                    <StackPanel>
                        <TextBlock Text="Current item DataContext:" />
                        <TextBlock FontWeight="Bold" Text="{Binding}" />

                        <TextBlock Text="Current parent (ExampleModel) DataContext:" />
                        <TextBlock FontWeight="Bold" Text="{utu:ItemsControlBinding Path=DataContext.SomeText}" />

                        <TextBlock Text="Accessing property of a parent ListView:" />
                        <TextBlock FontWeight="Bold" Text="{utu:ItemsControlBinding Path=Tag}" />

                        <TextBlock Text="Accessing property of a Page:" />
                        <TextBlock FontWeight="Bold" Text="{utu:AncestorBinding AncestorType=Page, Path=Tag}" />
                    </StackPanel>
                </DataTemplate>
            </ListView.ItemTemplate>
        </ListView>
    </Grid>
</Page>
```
we can forget the xmlnses... it was mentioned right above